torch/distributed/_sharding_spec/_internals.py

```diff
@@ -0,0 +1,22 @@
+import torch
+from torch.distributed.utils import _parse_remote_device
+
+def is_valid_device(device):
+    """
+    Checks if this is a valid remote/local device.
```

Comment:
Ni: Change the order of "remote/local" here to be consistent with the implementation?